src/official-account/official-account.ts

```diff
@@ -1,42 +1,46 @@
 /* eslint-disable camelcase */
 import { EventEmitter } from 'events'
+import { v4 } from 'uuid'
 
 import crypto from 'crypto'
-import { ContactGender, FileBox, log }  from 'wechaty-puppet'
+import { ContactGender, FileBox, log } from 'wechaty-puppet'
 import { FileBoxType } from 'file-box'
 
 import { normalizeFileBox } from './normalize-file-box'
 
 import {
   Webhook,
   VerifyArgs,
-}                      from './webhook'
+} from './webhook'
```

Comment:
Could we keep the spaces for better readability? Because I feel that each `import` should be a separate block visually.

Reply:
Could we keep the spaces for better readability? Because I feel that each import should be a separate block visually.